### DISCUSSION TITLE
RateLimit for FollowupMessageCreate

### DISCUSSION THREAD

**Discussion by wu56781234** [bwmarrin/discordgo#1332] [Q&A]
Since every interaction has unique token, every call of FollowupMessageCreate will create a bucket, and I didn't find a place to remove it. It seems this will cause out of memory eventually.
Any ideas?

**Comment by Vasyanhik** [comment]
> Since every interaction has unique token, every call of FollowupMessageCreate will create a bucket, and I didn't find a place to remove it. It seems this will cause out of memory eventually.
> Any ideas?
